### PR TITLE
Footer Resdesign

### PR DESCRIPTION
**Pull request by Thomaa04**
First commit for the new footer designs.

### CODE REVIEW

File: src/themes/uw-oshkosh-divi/style.scss [hunk 1]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+    height: 160px;
+  }
+
+  #footer-column-2 p{
+      width: 205px;
+  }
+
+  #footer-column-4{
+    img{
+      padding: 35px 14% 35px 0;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Is there a specific reason you are mixing pixels and percentages?

File: src/themes/uw-oshkosh-divi/style.scss [hunk 2]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+  padding: 10px 0 10px 0;
+  span{
+    padding: 0 10px;
   }
+}
 
-  h2 {
+#footer-body {
+  background-color: $light-grey;
+  padding: 45px 5% 45px 5%;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Reason for mixing percentages and pixels?

File: src/themes/uw-oshkosh-divi/style.scss [hunk 3]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
-    margin: 0 !important;
-    position: relative;
-    right: 20px;
-  }
-
-  h2 {
-    padding-bottom: 5px;
+#footer-address{
+  ul{
+    line-height: 1.75;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Should this have a unit associated with it?

File: src/themes/uw-oshkosh-divi/style.scss [hunk 4]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
-#footer-logos {
-  clear: both;
-  margin-bottom: 20px;
-  text-align: center;
-  white-space: nowrap;
-  width: 100%;
+#sitemap{
+  ul{
+    margin: 0;
+    line-height: 1.75;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: unit?

File: src/themes/uw-oshkosh-divi/style.scss [hunk 5]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+  background-color: $black;
+  padding: 0px;
 
-.app-icon {
-  height: 32px;
-  width: 32px;
-}
+  #footer-copy{
+    float: left;
+    color: #666766;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Hex value for variable should be stored in a SASS variable.

File: src/themes/uw-oshkosh-divi/style.scss [hunk 6]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+    font-size: 15px;
+    padding: 19px;
+  }
+  #footer-social{
+    float: right;
+    a{
+      display: table-cell;
+      text-align: center;
+      color: #666766;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Store color in variable

File: src/themes/uw-oshkosh-divi/style.scss [hunk 7]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+      padding: 15px;
+    }
 
-.wismap {
-  margin-right: 10px;
-  width: 124px;
-}
+    .fa-facebook:hover{
+      color: #3b5998;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Color should be stored in a variable

File: src/themes/uw-oshkosh-divi/style.scss [hunk 8]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+    }
 
-#address {
-  margin-top: 20px;
-}
+    .fa-twitter:hover{
+      color: #1da1f2;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Color should be stored in variable

File: src/themes/uw-oshkosh-divi/style.scss [hunk 9]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+    }
 
-#sitemap {
-  margin-right: 25px;
-  margin-top: 20px;
-}
+    .fa-instagram:hover{
+      color: #f56040;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Color should be stored in variable

File: src/themes/uw-oshkosh-divi/style.scss [hunk 10]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+    }
 
-#mobile-emergency {
-  float: left;
-  margin-top: 30px;
-  width: 170px;
-}
+    .fa-youtube-play:hover{
+      color: #cd201f;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Color should be stored in variable

File: src/themes/uw-oshkosh-divi/style.scss [hunk 11]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+    }
 
-#college-portrait {
-  margin-top: 61px;
-}
+    .fa-linkedin:hover{
+      color: #0077b5;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Color should be stored in variable

File: src/themes/uw-oshkosh-divi/style.scss [hunk 12]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+    }
 
-#footerSocialLinks {
-  margin-bottom: 20px;
+    .fa-pinterest-p:hover{
+      color: #bd081c;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Color should be stored in variable

File: src/themes/uw-oshkosh-divi/style.scss [hunk 13]
```diff
@@ -275,256 +276,243 @@ form.gsc-search-box {
...
+    }
 
-  li {
-    display: inline;
-    margin-right: 16px;
+    .fa-flickr:hover{
+      color: #8032B0;
```
Comment (musicaljoeker) [inline src/themes/uw-oshkosh-divi/style.scss]: Color should be stored in variable

File: src/themes/uw-oshkosh-divi/style.scss (hunk 3 above)
Comment (Thomaa04) [inline src/themes/uw-oshkosh-divi/style.scss]: The default is em, which I added.

**Pull request merged**